docs/usage.md

```diff
@@ -12,19 +12,35 @@ python run.py --help
 
 There are a few arguments that are required ocs test execution:
 
-* `--cred <cred_file>`
+* `--cred <cred_file>` - if you have aws configured by `aws configure`, see:
+    [AWS doc](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+    this parameter is not needed
 * `--suite <suite_file>`
 
 ## Useful Arguments
 
 Some non-required arguments that we end up using a lot:
 
 * `--log-level <level>` - set the log level that is output to stdout.
+* `--cluster-name <name>` - name of cluster.
+* `--cluster-path <path>` - path where to create the directory which will
+    contains all the installation/authentication information about cluster.
+    `Use this parameter when running on already deployed cluster!` You can
```

Comment:
for running on already deployed cluster, are we not using --reuse option

Reply:
@vavuthu reuse option is gone,  we can bring that when we are going to test with internal openstack and the purpose was not to create VMs and other resources again, I think the subtle difference here is  'use existing ocs cluster'  vs 'existing vm's and resources'